docs/guide/extending/custom_fieldtype_comparison.md

```diff
@@ -0,0 +1,117 @@
+# Creating custom version comparison of Field Types
+
+In the Back Office, you can compare the contents of a Field Type.
+Out of the box, you can compare versions of the TextLine Field Type (`ezstring`).
```

Comment:
This is true at _now_, but if all goes right, it will be outdated release, so not sure if we want to mention it now.